docs/http/http-spans.md

```diff
@@ -69,15 +69,11 @@ HTTP spans MUST follow the overall [guidelines for span names](https://github.co
 
 <!-- markdown-link-check-disable -->
 <!-- HTML anchors are not supported https://github.com/tcort/markdown-link-check/issues/225-->
-HTTP server span names SHOULD be `{method} {http.route}` if there is a
+HTTP server and client span names SHOULD be `{method} {http.route}` if there is a
 (low-cardinality) `http.route` available (see below for the exact definition of the [`{method}`](#method-placeholder) placeholder).
 
-If there is no (low-cardinality) `http.route` available, HTTP server span names
+If there is no (low-cardinality) `http.route` available, HTTP server and client span names
 SHOULD be [`{method}`](#method-placeholder).
```

Comment:
Since this would already change the client side span name, I'd suggest adding another fallback for it: in most client side HTTP instrumentations `http.route` won't be available, and span names with just `{method}` are not super useful.

Could we consider adding maybe `{method} {server.address}` as fallback if `http.route` is not available instead of just using `{method}` as span name?

Added this suggestion with more details here as well: https://github.com/open-telemetry/semantic-conventions/issues/923#issuecomment-2057192646

Reply:
let's take it separately from this PR. If span name changes are allowed, we should be able to revisit the default name as well, but that's another discussion.

Reply:
IIRC "method server.address" (or rather "method http.host"?) was already used at some point, but server.address does not meet the low cardinality requirement. In typical scenarios it will, but seeing that we went out of our way (very far even) to ensure even status code is really surely low cardinality, we have to be careful for server.address as well. E.g. using HTTP redirects to different subdomains, one could easily provoke a cardinality explosion.

Reply:
I agree this isn't about the core of the PR and we could move the discussion to #923. From whatever reason I don't have a resolve button, but whoever can, feel free to resolve this.

@Oberon00 we could continue it on #923. I'd be curious about the HTTP redirect + subdomains issue.